docs/pages/2020_Intro_Week/notebooks/bandstructure.ipynb

```diff
@@ -0,0 +1,190 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# A real world workchain example: electronic band structure"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%matplotlib inline\n",
+    "%aiida\n",
+    "\n",
+    "from datetime import datetime, timedelta\n",
+    "from aiida.engine import run\n",
+    "from aiida.plugins import DbImporterFactory\n",
+    "CodDbImporter = DbImporterFactory('cod')\n",
+    "\n",
+    "PwBandStructureWorkChain = WorkflowFactory('quantumespresso.pw.band_structure')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Calculating the electronic band structure with an AiiDA workchain\n",
+    "This tutorial will show how useful a workchain can be in performing a well defined task, such as computing and visualizing the electronic band structure for a simple crystal structure. The goal of this tutorial is not to show you the intricacies of the actual workchain itself, but rather to serve as an example that workchains can simplify standard workflows in computational materials science. The workchain that we will use here will employ Quantum ESPRESSO's `pw.x` code to calculate the charge densities for several crystal structures and compute a band structure from those. Many choices that normally face the researcher before being able to perform this calculation, such as the selection of suitable pseudo potentials, energy cutoff values, k-point grids and k-point paths along high symmetry points, are now performed automatically by the workchain. All that remains for the user to do is to simply define a structure, pass it to the workchain and sit back!"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Below, we import the crystal structure of Al (aluminium) as an example, and run the `PwBandStructureWorkChain` for that structure. The estimated run time is noted in a comment in the calculation cell."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Loading the COD importer so we can directly import structure from COD id's\n",
+    "importer = CodDbImporter()\n",
+    "# Make sure here to define the correct codename that corresponds to the pw.x code installed on your machine of choice\n",
+    "codename = 'qe-6.5-pw@localhost'\n",
```

Comment:
"qe-6.5-pw@localhost" is available on the generic profile, but not on the profile they will have created using the perovskite archive. Maybe we should mention that, or use `%aiida generic`

Reply:
Thanks. I think we should have used a single profile: #264 anyway thanks for the hint, I'll fix this

Reply:
I put notes at the top on which profile they should be using. We'll remove the notes when #264 is fixed